### PR TITLE
CHANGE folder structure && CHANGE package-names

### PR DESCRIPTION
**Pull request by ndelangen**
## What I did
I implemented the proposed changes to the folder structure and package names

### CODE REVIEW

**Review by shilman — CHANGES_REQUESTED** [review]
@ndelangen when i try to test it, I get this error:

```
lerna ERR! execute npm ERR! 404 Not found : @storybook/storybook-database-cloud
lerna ERR! execute npm ERR! 404 
lerna ERR! execute npm ERR! 404  '@storybook/storybook-database-cloud' is not in the npm registry.
lerna ERR! execute npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
```

It is referenced by `addon-comments`. I think we should either migrate `database-*` to the monorepo, revert to `@kadira/database-cloud`, or remove `addon-comments` from the monorepo for now.

File: addons/actions/README.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ Import the `action` function and use it to create actions handlers. When creatin
 > *Note: Make sure NOT to use reserved words as function names. [issues#29](https://github.com/storybooks/storybook-addon-actions/issues/29#issuecomment-288274794)*
 
 ```js
-import { storiesOf, action } from '@storybook/storybook'
-// or import { action } from '@storybook/storybook-addon-actions'
+import { storiesOf, action } from '@storybook/react'
```
Comment (shilman) [inline addons/actions/README.md]: FYI all of these are going to conflict with the changes I made in https://github.com/storybooks/storybook/issues/1017

**Comment by UsulPro** [comment]
Thanks, @ndelangen for this rearrangement! 👍 

Some points to discuss:

- rename `decorator-centered` to `addon-centered` since this is also an addon. (We don't have special names for panels, storiesOf extensions).

- move `storyshots` to addons folder. Since we import it additionally we can call it "addon"

- changing `packages` to `lib` looks good to me, but maybe `core` also could fit?

- what is the purpose of the `app` folder? Maybe it should be located in the `examples\` folder?

**Comment by ndelangen** [comment]
App folder are the what previously were @kadira/storybook and @kadira/storybook-react-native

In the future there may be other UI libraries/frameworks there too.

File: addons/actions/README.md (hunk 1 above)
Comment (ndelangen) [inline addons/actions/README.md]: Yup I'm correcting for that.

**Review by shilman — APPROVED** [review]

**Comment by shilman** [comment]
I agree with @UsulPro 's comments above. Otherwise, the changes look good to me. Thanks for all your work on getting this cleaned up!!

**Comment by tmeasday** [comment]
Do we want to change `storyshots` to `@storybook/storyshots`?

**Comment by ndelangen** [comment]
I just want to get this finished and merged, so just shout how you want it now, or forever hold your peace 🙊

**Review by aaronmcadam — APPROVED** [review]

**Comment by ndelangen** [comment]
I will continue work on the other branch!

**Pull request merged**